src/cljstyle/main.clj

```diff
@@ -14,7 +14,15 @@
    [nil  "--stats FILE" "Write formatting stats to the named file. The extension controls the format and may be either 'edn' or 'tsv'."]
    [nil  "--no-color" "Don't output ANSI color codes."]
    ["-v" "--verbose" "Print detailed debugging output."]
-   ["-h" "--help" "Show help and usage information."]])
+   ["-h" "--help" "Show help and usage information."]
+   {:id :exclude-dirs
+    :long-opt "--exclude-dir"
+    :required "GLOB"
+    :desc "A directory glob to exclude from styling. May be set multiple times."
+    :default #{}
```

Comment:
Specifying a default string in addition to this would be nice too - that way we're not printing an empty set in the help text.